benchmark/largerandom/sax.h

```diff
@@ -43,15 +43,31 @@ struct sax_point_reader_visitor {
     return SUCCESS;
   }
   simdjson_really_inline error_code visit_primitive(json_iterator &, const uint8_t *value) {
-    return numberparsing::parse_double(value).get(buffer[idx++]);
+    if(idx == 4) { return simdjson::SUCCESS; }
```

Comment:
Or is this actually an error (do we expect to allow multiple fields)?

Reply:
In this (not very good) code the value "4" is a codeword for "not x or y or z".

Reply:
Let me tweak the code so that this is nicer looking. One minute...

Reply:
There you go. I changed the code with an enum so that the semantics is clearer. It is still not good code, but we are not going to advertise the SAX anyhow...